server/src/grpc/server.rs

```diff
@@ -23,143 +26,182 @@ pub struct ServerImpl {}
+            grpc_error!("error updating nodes: {}", e);
+            Status::internal(e.to_string())
+        })?;
+
+        Ok(Response::new(grpc_server::UpdateResponse { updated: true }))
     }
 
-    #[cfg(not(tarpaulin_include))]
     async fn update_zones(
         &self,
         request: Request<grpc_server::UpdateZonesRequest>,
     ) -> Result<Response<grpc_server::UpdateResponse>, Status> {
-        grpc_debug!("(update_zones) entry.");
+        grpc_debug!("entry.");
 
         // Update nodes in PostGIS
-        match zone::update_zones(request.into_inner().zones).await {
-            Ok(_) => Ok(Response::new(grpc_server::UpdateResponse { updated: true })),
-            Err(e) => {
-                grpc_error!("(update_zones) error updating zones: {}", e);
-                Err(Status::internal(e.to_string()))
-            }
-        }
+        let zones = request.into_inner().zones;
+        zone::update_zones(zones).await.map_err(|e| {
+            grpc_error!("error updating zones: {}", e);
+            Status::internal(e.to_string())
+        })?;
+
+        Ok(Response::new(grpc_server::UpdateResponse { updated: true }))
     }
 
-    #[cfg(not(tarpaulin_include))]
     async fn update_flight_path(
         &self,
         request: Request<grpc_server::UpdateFlightPathRequest>,
     ) -> Result<Response<grpc_server::UpdateResponse>, Status> {
-        grpc_debug!("(update_flight_path) entry.");
+        grpc_debug!("entry.");
 
         // Update nodes in PostGIS
-        match flight::update_flight_path(request.into_inner()).await {
-            Ok(_) => Ok(Response::new(grpc_server::UpdateResponse { updated: true })),
-            Err(e) => {
-                grpc_error!("(update_flight_path) error updating flight path: {}", e);
-                Err(Status::internal(e.to_string()))
-            }
-        }
+        let request = request.into_inner();
+        flight::update_flight_path(request).await.map_err(|e| {
+            grpc_error!("error updating flight path: {}", e);
+            Status::internal(e.to_string())
+        })?;
+
+        Ok(Response::new(grpc_server::UpdateResponse { updated: true }))
     }
 
-    #[cfg(not(tarpaulin_include))]
     async fn best_path(
         &self,
         request: Request<grpc_server::BestPathRequest>,
     ) -> Result<Response<grpc_server::BestPathResponse>, Status> {
-        grpc_debug!("(best_path) entry.");
+        grpc_debug!("entry.");
         let request = request.into_inner();
-        match best_path::best_path(request).await {
-            Ok(paths) => {
-                let response = grpc_server::BestPathResponse { paths };
-                Ok(Response::new(response))
-            }
-            Err(e) => {
-                grpc_error!("(best_path) error getting best path: {}", e);
-                Err(Status::internal(e.to_string()))
+
+        let paths = best_path::best_path(request).await.map_err(|e| {
+            grpc_error!("error getting best path.");
```

Comment:
Add error message output to error line as well:
`grpc_error!("error getting best path: {}", e);`

Reply:
568a2a762168b9f1fe611860989da50cc02c494a